### DISCUSSION TITLE
How to Donate ❤️

### DISCUSSION THREAD

**Discussion by coccoinomane** [coccoinomane/crabada.py#57] [General]
Hello snib-snibbers :-)

The donation mechanism is very simple and ***completely optional***:

1. Specifiy a `DONATE_PERCENTAGE` value in your _.env_, for example `DONATE_PERCENTAGE=5%`.
2. Crabada.py will automatically donate that percentage of your TUS and CRA rewards.
3. The donation will cost a negligible amount of gas, less than 1% of what you already spend for mining or looting!

I understand that automatic donations are a bit scary, therefore I am happy to answer any doubt here 🙂

Cheers,
Cocco

**Comment by coccoinomane** [comment]
PS: @yigitest would you mind hitting me on Discord? My handle is coccoinomane, the number code is 0676, thanks!

**Reply by yigitest** [reply]
Your account is closed to friend requests, You add me instead yigitest#8554.

**Comment by Mrooie** [comment]
Yay!! Finally, 'cause you deserve it!
